src/bloaty.h

```diff
@@ -500,18 +504,18 @@ struct RollupOutput {
   bool diff_mode_ = false;
 
   void PrettyPrint(size_t max_label_len, std::ostream* out) const;
-  void PrintToCSV(std::ostream* out) const;
+  void PrintToCSV(std::ostream* out, bool tabs=false) const;
```

Comment:
Bloaty follows the Google C++ style guide, which generally disallows default arguments. Please use explicit parameters only.